Formula/fish.rb

```diff
@@ -22,9 +22,16 @@ class Fish < Formula
 
   def install
     system "autoconf" if build.head? || build.devel?
+    args = %W[
+      --prefix=#{prefix}
+      --with-extra-functionsdir=#{HOMEBREW_PREFIX}/share/fish/vendor_functions.d
```

Comment:
is it acceptable to modify Makefile (using `inreplace` or something else) inside Formula?
Im not sure if upstream will remove directory installation from Makefile.